src/Blorc.OpenIdConnect.DemoApp/Program.cs

```diff
@@ -44,6 +44,20 @@
         options.Scope = "openid profile demo-api";
         options.TimeForUserInactivityAutomaticSignOut = 15000;
         options.TimeForUserInactivityNotification = 10000;
+        /* Options below are not required */
+
+        // The resource parameter that will be sent to the auth request
+        options.Resource = "https://your-api.com";
```

Comment:
Let's not add this as example code, but as comment

Reply:
Agree, remove this from demo program and update the readme.md with the documentation.

Reply:
Sure. I'm in the hospital, will be back in a few days.

Reply:
@GeertvanHorrik the demo app has been updated. Can you let me know which part of docs should be updated?